stage #2.5/module #4 REST Basics - Spring Web/module_description.md

```diff
@@ -1 +1,12 @@
-# REST Basics - Spring Web
\ No newline at end of file
+# REST Basics - Spring Web
+
+This module provides basic patterns, principals and approaches that can be applied for designing REST API.
```

Comment:
'...basic patterns, principals and approaches...' - 'principals' should be changed to 'principles'